### DISCUSSION TITLE
Are artists supposed to be automatically imported from "Liked" artists in Deezer, Tidal, Spotify, etc?

### DISCUSSION THREAD

**Discussion by trashdad** [RandomNinjaAtk/docker-lidarr-extended#111] [General]
Despite running Deemix along with DLE, linking Tidal, but also adding a Spotify list to Lidarr I cannot get my "liked" artists to import automatically in Lidarr. Adding them manually is a huge chore as waiting for the manually typed search can take time. The search must be exact to work well. It does not take MBID like it says it should. Is there a better way?

**Comment by RandomUser345** [comment]
What works for me is "following" the artist on Spotify and import the "Spotify Followed Artists" List in Lidarr.

**Reply by trashdad** [reply]
This is what I am trying to do, but there is some sort of error. I've asked in the Servarr Discord and this is a snippet of the conversation:

After I provided my trace logs:

```

2022-10-27 00:16:24.8|Trace|Http|Req: 36 [GET] /api/v1/queue?page=1&pagesize=1000000000&sortDirection=descending&sortKey=progress&includeUnknownArtistItems=true&apikey=(removed) (from 127.0.0.1 curl/7.80.0)
Well that's just plain invalid
The page size is a 250 limit I believe, so whatever you're having trying to get all that.... 1) don't and 2) makes no sense
I don't see anything to do with a list in there?
Seems step 3 was skipped..?


No....it's curl from the logs. Nothing to do with in-app and you cannot even set that size that high in app..?

Not a lidarr thing. You have something with the useragent of curl that you configured to hit Lidarr's api  with that ridiculous page size
```

**Reply by RandomNinjaAtk** [reply]
Your trace log is a entirely different issue and not related to your question.

**Reply by trashdad** [reply]
Fair enough. Just trying to learn how to ask the right questions in the right places. Always happy to be provided some direction so I don't waste anyone's time.

**Comment by RandomNinjaAtk** [comment]
This script does not have a function to import liked artists. It can automatically add artists that are "related/similar" to your existing artists.
